CHANGES

```diff
@@ -22,6 +22,8 @@ Features added
 Bugs fixed
 ----------
 
+* #8588: autodoc: autodoc_type_aliases does not support dotted name
```

Comment:
How about describing this as an improvement? It's easier to understand the change.
```suggestion
* #8588: autodoc: autodoc_type_aliases now supports dotted name. It allows you to define an alias for a class with module name like ``foo.bar.BazClass``.
```